app/models/carrier.rb

```diff
@@ -18,6 +18,19 @@ class Carrier < ApplicationRecord
 
   has_many_attached :photos
 
+  enum status: {
+    available: 0,
+    unavailable: 1,
+    disabled: 2,
```

Comment:
What is 3 😄 ?

Reply:
C was my first programming language and one of the weird little accents I've kept since then is using powers of two for things like enums. If you did it that way in C you could use bit masking to store enum values in a single byte.

I'd be happy to change it if we think it'll be confusing.

Reply:
Yeah I think it is a little bit confusing, I would stick with (0,1,2,3)